WEB/Src/PerformanceCollector/Perf.Tests/QuickPulse/QuickPulseServiceClientTests.cs

```diff
@@ -1187,10 +1215,12 @@ public void QuickPulseServiceClientSubmitsInstanceNameToServiceWithPing()
         {
```

Comment:
QuickPulseServiceClientSubmitsInstanceNameToServiceWithPing now checks both instance and role, let's rename the test to reflect its new purpose.